src/maptm/kconfig/Kconfig.managers

```diff
@@ -0,0 +1,13 @@
+menuconfig MANAGER_MAPTM
```

Comment:
Add `src/maptm/kconfig/Kconfig.managers` to `kconfig/Kconfig.managers`, just after `nfm`:
* https://github.com/plume-design/opensync/blob/master/kconfig/Kconfig.managers#L19